java/conf/default/rhn_hibernate.conf.SUSE

```diff
@@ -35,3 +35,7 @@ hibernate.jdbc.batch_size=0
 hibernate.cache.provider_class=org.hibernate.cache.EhCacheProvider
 hibernate.cache.region.factory_class=org.hibernate.cache.ehcache.SingletonEhCacheRegionFactory
 hibernate.id.new_generator_mappings = false
+<<<<<<< HEAD
```

Comment:
You need to fix this conflict before I can merge.

Reply:
Didn't see that this was there, thanks, fixed